contracts/campaignAirdrops/ExistingERC20DropCampaign.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.14;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "../interfaces/AirdropMerkleProof.sol";
+import "../interfaces/IAirBroFactory.sol";
+
+/// @title Airdrops existing ERC20 tokens for airdrop recipients
+contract ExistingERC20DropCampaign is AirdropMerkleProof {
+    using SafeERC20 for IERC20;
+
+    IERC20 public immutable rewardToken;
+    address public immutable airbroCampaignFactoryAddress;
+    uint256 public immutable tokenSupply;
+
+    string public airdropType = "ERC20";
```

Comment:
const / remove
Idk what is the purpose of this hardcoded string

Reply:
It's basically a way for users to know what the reward will be. Inherited from previous developer so we kept it

Reply:
we can change it to constant then, as it won't change